src/phoenix/server/api/routers/v1/evaluations.py

```diff
@@ -130,21 +135,36 @@ async def get_evaluations(request: Request) -> Response:
             _read_sql_span_evaluations_into_dataframe,
             project_name,
         )
-    if span_evals_dataframe.empty:
+        trace_evals_dataframe = await connection.run_sync(
+            _read_sql_trace_evaluations_into_dataframe,
+            project_name,
+        )
+        document_evals_dataframe = await connection.run_sync(
+            _read_sql_document_evaluations_into_dataframe,
+            project_name,
+        )
+    if trace_evals_dataframe.empty and span_evals_dataframe.empty and not document_evals_dataframe:
         return Response(status_code=HTTP_404_NOT_FOUND)
 
-    loop = asyncio.get_running_loop()
-
-    async def content() -> AsyncIterator[bytes]:
-        for eval_name, span_evals_dataframe_for_name in span_evals_dataframe.groupby(
-            "name", as_index=False
-        ):
-            span_evals = SpanEvaluations(str(eval_name), span_evals_dataframe_for_name)
-            yield await loop.run_in_executor(
-                None, lambda: table_to_bytes(span_evals.to_pyarrow_table())
-            )
```

Comment:
Starlette will ensure that this is non-blocking, even if we pass in a synchronous stream to `content`.

https://github.com/encode/starlette/blob/eb76cae6fdb6c1b0bfcace17d0dec946fe767f84/starlette/responses.py#L230